core/modules/node/node.types.inc

```diff
@@ -408,26 +413,48 @@ function node_type_form($form, &$form_state, $type = NULL) {
   $form['permissions']['permissions'] = $content_type_permissions;
 
   // Revision settings.
+  if (user_access('administer permissions')) {
+    $revisions_help = t('Revisions allow content editors with <a href="@permissions-url" target="_blank">appropriate permissions</a> to view changes made to content over time, and revert them if needed.',
+      array('@permissions-url' => url('admin/config/people/permissions', array('query' => array('search' => 'revision')))));
+  }
+  else {
+    $revisions_help = t('Revisions allow content editors with appropriate permissions to view changes made to content over time, and revert them if needed.');
+  }
   $form['revision'] = array(
     '#type' => 'fieldset',
     '#title' => t('Revision settings'),
     '#collapsible' => TRUE,
     '#collapsed' => TRUE,
     '#group' => 'additional_settings',
     '#weight' => 10,
+    '#description' => $revisions_help,
   );
-  $form['revision']['revision']['revision_enabled'] = array(
-    '#type' => 'checkbox',
-    '#title' => t('Show option to create new revisions'),
-    '#default_value' => $type->settings['revision_enabled'],
-    '#parents' => array('revision_enabled'),
-    '#description' => t('Revisions allow content editors to view changes over time and revert changes if needed.'),
+  $form['revision']['revision']['revisions'] = array(
```

Comment:
Line 786 has another 'revision_enabled' that should be changed to 'revisions'.

Reply:
Good catch @kswan 👍🏼 

Fixed with [`3f8398e` (#4546)](https://github.com/backdrop/backdrop/pull/4546/commits/3f8398e7e6fed31cab9d2fb0f1890fcd48d5d1a4)